### PR TITLE
Templatize Campaigns

### PR DESCRIPTION
**Pull request by JLLeitschuh**
This PR extracts the PR Body, and Commit Message into reusable template files.
This allows new campaigns to get added with less overhead, and messaging can be
configured and changed in a common location.

Signed-off-by: Jonathan Leitschuh <Jonathan.Leitschuh@gmail.com>

### CODE REVIEW

File: src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This leaves your build vulnerable to allowing a [Man in the Middle](https://en.w
 This vulnerability has a CVSS v3.0 Base Score of [8.1/10](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H).
 
 [POC code](https://max.computer/blog/how-to-take-over-the-computer-of-any-java-or-clojure-or-scala-developer/) has existed since 2014 to maliciously compromise a JAR file in-flight.
-MITM attacks against HTTP are [increasingly common](https://security.stackexchange.com/a/12050), for example [Comcast is known to have done it to their own users](https://thenextweb.com/insights/2017/12/11/comcast-continues-to-inject-its-own-code-into-websites-you-visit/#).
+MITM attacks against HTTP are [increasingly common](https://security.stackexchange.com/a/12050), for example [Comcast is known to have done it to their own users](https://thenextweb.com/news/comcast-continues-to-inject-its-own-code-into-websites-you-visit).
 
 This contribution is a part of a submission to the [GitHub Security Lab](https://securitylab.github.com/) Bug Bounty program.
```
Comment (scovetta) [inline src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md:17]: Is this needed?

**Review by scovetta — APPROVED** [review]

File: src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md (hunk 1 above)
Comment (JLLeitschuh) [reply src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md:17]: You'd be surprised. I've gotten arguments from some that it's not a common attack vector.

Comment (scovetta) [reply src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md:17]: Sorry, I just means the GitHub Bug Bounty program reference. :-)

Comment (JLLeitschuh) [reply src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md:17]: Oh, yes, that. I should can that now. It's not relevant anymore.

Comment (JLLeitschuh) [reply src/omega_moderne_client/campaign/campaigns/http_in_maven_pom/pr_message.md]: Resolved! Much appreciated!

File: src/omega_moderne_client/campaign/campaigns/pr_message_footer.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   Note: Please include `OpenSSF: Omega Disclosure` in the subject of your email, so it is not missed.
+
+## Why didn't you disclose privately (ie. coordinated disclosure)?
+
+This PR was automatically generated, in-bulk, and sent to this project as well as many others, all at the same time.
+
+This is technically what is called a "Full Disclosure" in vulnerability disclosure, and I agree it's less than ideal.
+If GitHub offered a way to create private pull requests to submit pull requests, I'd leverage it, but that infrastructure, sadly, doesn't exist yet.
+
+The problem is that, as an open source software security researcher, I (exactly like open source maintainers), I only have so much time in a day.
```
Comment (bobcallaway) [inline src/omega_moderne_client/campaign/campaigns/pr_message_footer.md:29]:
```suggestion
The problem is that, as an open source software security researcher, I (exactly like open source maintainers), only have so much time in a day.
```

**Comment by brianbehlendorf** [comment]
Here's the text for a proposed web page to provide context, extracted from the markdown in this commit, let's iterate here:

https://docs.google.com/document/d/1YmcDzUeL0V0Tu3yHlh2NUe6MLSznYqQc4Vn1gkPS8GE/edit#heading=h.z2eykpymhfvl

**Comment by JLLeitschuh** [comment]
I'm going to merge this. We can continue the discussion of the proposed text and apply it in a later pull request

**Pull request merged**